client/src/components/LoginForm/LoginForm.spec.js

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from '@testing-library/react'
+
+import LoginForm from './LoginForm'
+
+describe('LoginForm', () => {
+  it('renders', () => {
+    render(<LoginForm />)
+
+    const h2Element = screen.getAllByRole('heading', { level: 2 })
+    expect(h2Element).toHaveLength(1)
+
+    const textUsername = screen.getByLabelText('Username')
+    expect(textUsername).toBeInTheDocument()
+
+    const textPassword = screen.getByLabelText('Password')
+    expect(textPassword).toBeInTheDocument()
+
+    const buttonElement = screen.getAllByRole('button')
+    expect(buttonElement).toHaveLength(1)
```

Comment:
Does this test really makes sense? Would it as alternative better to just test if the button is in the document or why it is important that there is only one button?

Reply:
The more specific your tests are the better